packages/components/pie-toast/src/defs.ts

```diff
@@ -44,6 +46,13 @@ export interface ToastProps {
    * The leading action for the toast.
    */
   leadingAction?: ActionProps;
+
+  /**
+   * It set the duration of the toast in milliseconds before it auto-dismiss
+   * If the value is null auto-dismiss is disabled
+   * If the value is not provided it auto-dismiss after 5 seconds (5000 milliseconds)
+   */
+  duration?: number | null;
```

Comment:
We don't tend to use `null` very often. What do you think about having `0` as the value for persistent toasts?

Reply:
@xander-marjoram Interesting point. I will think about it with @kevinrodrigues

Reply:
Discussed with @thejfreitas.

@xander-marjoram when refining this ticket there were some comments around using `null` as opposed to `0` which I think i used in the first place but cant remember. @raoufswe also made a suggested to use `null` too we looked at this UI library as part of that. https://v2.chakra-ui.com/docs/components/toast (`null` representing `infinite` as opposed to using `0`).